fs/procfs/fs_procfs.c

```diff
@@ -186,25 +176,21 @@ static const struct procfs_entry_s g_procfs_entries[] =
 #endif
 };
 
-#ifdef CONFIG_FS_PROCFS_REGISTER
 static const uint8_t g_base_entrycount = sizeof(g_base_entries) /
                                          sizeof(struct procfs_entry_s);
 
+#ifdef CONFIG_FS_PROCFS_REGISTER
 static FAR struct procfs_entry_s *g_procfs_entries;
 static uint8_t g_procfs_entrycount;
 #else
-static const uint8_t g_procfs_entrycount = sizeof(g_procfs_entries) /
-                                           sizeof(struct procfs_entry_s);
+static FAR const struct procfs_entry_s *g_procfs_entries = g_base_entries;
```

Comment:
Maybe we can use `define` and not introduce new variable?

Reply:
Done.